client/me/purchases/manage-purchase/index.jsx

```diff
@@ -593,6 +596,44 @@ class ManagePurchase extends Component {
 		);
 	}
 
+	handleReinstall = async () => {
+		this.setState( { isReinstalling: true } );
+		const siteId = this.props.purchase.siteId;
+		try {
+			const response = await reinstallPlugins( siteId );
+
+			this.props.successNotice( response.message, { duration: 5000 } );
+		} catch ( error ) {
+			this.props.errorNotice( error.message );
+		} finally {
+			this.setState( { isReinstalling: false } );
+		}
+	};
+
+	renderReinstall() {
+		const { purchase, productsList, translate } = this.props;
+		const { isReinstalling } = this.state;
+		if ( ! ( purchase.active && hasMarketplaceProduct( productsList, purchase.productSlug ) ) ) {
+			return null;
+		}
+
+		return (
+			<CompactCard tagName="a" onClick={ isReinstalling ? null : this.handleReinstall }>
+				{ isReinstalling ? (
+					<>
+						<Spinner className="card__icon" />
+						{ translate( 'Reinstalling' ) }...
```

Comment:
It might not matter but I suggest that you put the ellipses (`…`) inside the translated string [like we do here](https://github.com/Automattic/wp-calypso/blob/d05dbbe45e09c112baad817541945124f7e6555f/client/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-submit-button.tsx#L48) and [here](https://github.com/Automattic/wp-calypso/blob/d05dbbe45e09c112baad817541945124f7e6555f/client/jetpack-connect/site-url-input.jsx#L76) and [here](https://github.com/Automattic/wp-calypso/blob/d05dbbe45e09c112baad817541945124f7e6555f/client/jetpack-connect/remote-credentials.js#L263). You may also want to use the standalone `…` character (option-semicolon on the en-US MacOS keyboard) instead of three dots `...`.

Reply:
That was my first try, but then I discovered the [`i18n-ellipsis` linting rule](https://github.com/Automattic/wp-calypso/blob/add/plugin-purchase-reinstall/packages/eslint-plugin-wpcalypso/README.md#supported-rules). I have also tried with the `…` character (thanks for the keyboard shortcut 😄) and it also triggers the linting error.

Reply:
TIL! 🤔 although that [rule's description](https://github.com/Automattic/wp-calypso/blob/add/plugin-purchase-reinstall/packages/eslint-plugin-wpcalypso/docs/rules/i18n-ellipsis.md) says explicitly that using the … character is _not_ a warning. Maybe try it again? If it still gives an error, we might want to ping @sirreal.

Reply:
I have tried again and as you can see by last commit [9701076](https://github.com/Automattic/wp-calypso/pull/72662/commits/9701076387f9185c8dce2d6de6d62796fc539b8e) it has worked fine (maybe I didn't save?). Anyways, I know now that rule and I will use the `…` character when required. Thanks for the suggestion!